rst2pdf/style-to-yaml.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+'''
```

Comment:
style nit: could we use double quotes here? That's more typical for multi-line docstrings.

Reply:
Fixed, but half the project uses single ones - decided it made no sense to fix other files in this PR. One for the automated stylechecker tools and a rainy day, perhaps